### PR TITLE
Allow the stats timeout to be user-configurable

### PR DESCRIPTION
**Pull request by dubeyShivank**

### CODE REVIEW

File: packages/millicast-sdk/src/PeerConnection.js [hunk 1]
```diff
@@ -43,13 +43,13 @@ export default class PeerConnection extends EventEmitter {
...
    */
-  async createRTCPeer (config = { autoInitStats: true }) {
-    logger.info('Creating new RTCPeerConnection')
-    logger.debug('RTC configuration provided by user: ', config)
+  async createRTCPeer (config = { autoInitStats: true, statsIntervalMs: 1000 }) {
+    logger.info('Creating new RTCPeerConnection', config)
     this.peer = instanceRTCPeerConnection(this, config)
     if (config.autoInitStats) {
-      this.initStats()
+      this.initStats(config.autoInitStats, config.statsIntervalMs)
```
Comment (Santiago-Souto) [inline packages/millicast-sdk/src/PeerConnection.js:52]: Couldn't we pass in config object and then extract only the relevant options. Mainly thinking about scalability

File: packages/millicast-sdk/src/PeerConnection.js [hunk 2]
```diff
@@ -336,11 +336,11 @@ export default class PeerConnection extends EventEmitter {
    *   console.log('Stats from event: ', stats)
    * })
    */
-  initStats () {
+  initStats (autoInitStats, statsIntervalMs) {
```
Comment (Santiago-Souto) [inline packages/millicast-sdk/src/PeerConnection.js:339]: Same here, could take something like `options`

File: packages/millicast-sdk/src/PeerConnection.js [hunk 3]
```diff
@@ -336,11 +336,11 @@ export default class PeerConnection extends EventEmitter {
...
     if (this.peerConnectionStats) {
       logger.warn('PeerConnection.initStats() has already been called.  Automatic initialization occurs via View.connect(), Publish.connect() or this.createRTCPeer(). See options')
     } else if (this.peer) {
-      this.peerConnectionStats = new PeerConnectionStats(this.peer)
+      this.peerConnectionStats = new PeerConnectionStats(this.peer, { autoInitStats, statsIntervalMs })
```
Comment (Santiago-Souto) [inline packages/millicast-sdk/src/PeerConnection.js:343]: `options` as well, and in the constructor extract it

File: packages/millicast-sdk/src/PeerConnection.js (hunk 1 above)
Comment (dubeyShivank) [reply packages/millicast-sdk/src/PeerConnection.js]: Yes we can and now we do :)

File: packages/millicast-sdk/src/PeerConnection.js (hunk 2 above)
Comment (dubeyShivank) [reply packages/millicast-sdk/src/PeerConnection.js]: Yep

File: packages/millicast-sdk/src/PeerConnection.js (hunk 3 above)
Comment (dubeyShivank) [reply packages/millicast-sdk/src/PeerConnection.js]: Si senor

**Comment by Santiago-Souto** [comment]
@dubeyShivank are we going to save both changeset files? They look similar

**Comment by dubeyShivank** [comment]
> @dubeyShivank are we going to save both changeset files? They look similar

oh, I'll delete one! Thank you! I saw the `no changesets detected` message so I added a new one today. Mea culpa!

**Review by Santiago-Souto — APPROVED** [review]
lgtm

**Pull request merged**